src/nvme/util.c

```diff
@@ -476,3 +476,27 @@ int nvme_get_directive_receive_length(enum nvme_directive_dtype dtype,
 		return -EINVAL;
 	}
 }
+
+static const char * const libnvme_status[] = {
+	[ENVME_CONNECT_RESOLVE] = "failed to resolve host",
+	[ENVME_CONNECT_ADDRFAM] = "unrecognized address family",
+	[ENVME_CONNECT_TRADDR] = "failed to get traddr",
+	[ENVME_CONNECT_TARG] = "need a transport (-t) argument",
+	[ENVME_CONNECT_AARG] = "need a address (-a) argument\n",
```

Comment:
For a pure library use, no commandline is necessarily involved and the `(-t)` and `(-a)` switches might be misleading. These are the strings that will potentially be presented to end user in a GUI.

Reply:
Good point. And maybe it should be rephrased to "missing transport argument" etc. I've just copied the strings from the nmve_log() entries. Was a bit lazy :)

Reply:
Fixed now.